### PR TITLE
Minor improvement to microcircuit example

### PR DESCRIPTION
**Pull request by morales-gregorio**
As mentioned in #1207 the examples code has already improved significantly and I could only identify one place where a warning would be fitting. 

Additionally I spotted one superfluous import.

Best,
Aitor

### CODE REVIEW

**Comment by morales-gregorio** [comment]
I have changed the problematic files from #1208 in this PR too. The PBS part was removed and writing the submission file is something the user will have to do own their own.

File: pynest/examples/Potjans_2014/helpers.py [hunk 1]
```diff
@@ -293,7 +292,8 @@ def firing_rates(path, name, begin, end):
         # zeros are included
         all_mean_rates.append(np.mean(rate_per_neuron))
         all_std_rates.append(np.std(rate_per_neuron))
-    print('Mean rates: {} spikes/s'.format(np.around(all_mean_rates, decimals=3)))
+    print('Mean rates: {} spikes/s'.format(np.around(all_mean_rates,
+                                                     decimals=3)))
```
Comment (stinebuu) [inline pynest/examples/Potjans_2014/helpers.py:296]: Could you revert back to one line for this? We allow lines of 120 characters in NEST.

File: pynest/examples/Potjans_2014/network.py [hunk 2]
```diff
@@ -210,7 +211,8 @@ def __derive_parameters(self):
             DC_amp = np.zeros(self.num_pops)
         else:
             if nest.Rank() == 0:
-                print('DC input compensates for missing Poisson input.\n')
+                warnings.warn('DC input created to compensate missing Poisson'
+                              'input.\n')
```
Comment (stinebuu) [inline pynest/examples/Potjans_2014/network.py:215]: This could also be one line.

**Review by stinebuu — COMMENTED** [review]
Thanks! I have two small formatting comments, but otherwise it looks good :)

File: pynest/examples/Potjans_2014/helpers.py (hunk 1 above)
Comment (morales-gregorio) [reply pynest/examples/Potjans_2014/helpers.py]: Done

File: pynest/examples/Potjans_2014/network.py (hunk 2 above)
Comment (morales-gregorio) [reply pynest/examples/Potjans_2014/network.py]: Done

**Review by stinebuu — APPROVED** [review]
Thanks!

**Comment by stinebuu** [comment]
@morales-gregorio Just noticed that GitHub actions complains about some pep8 formatting errors in `network.py`. Could you check that? If it is the line break I made you revert I apologize! I did not check how long the line was before I asked you to put it in one line.

**Comment by morales-gregorio** [comment]
Hi @stinebuu I dsee all the PEP8 errors refer to line breaks after a binary operator:
```
MSGBLD0195: [PEP8] pynest/examples/Potjans_2014/network.py:193:72: W504 line break after binary operator
MSGBLD0195: [PEP8] pynest/examples/Potjans_2014/network.py:195:57: W504 line break after binary operator
MSGBLD0195: [PEP8] pynest/examples/Potjans_2014/network.py:196:66: W504 line break after binary operator
MSGBLD0195: [PEP8] pynest/examples/Potjans_2014/network.py:198:63: W504 line break after binary operator
MSGBLD0195: [PEP8] pynest/examples/Potjans_2014/network.py:329:46: W504 line break after binary operator
MSGBLD0195: [PEP8] pynest/examples/Potjans_2014/network.py:423:56: W504 line break after binary operator
MSGBLD0195: [PEP8] pynest/examples/Potjans_2014/network.py:452:71: W504 line break after binary operator
MSGBLD0195: [PEP8] pynest/examples/Potjans_2014/network.py:459:79: W504 line break after binary operator
MSGBLD0195: [PEP8] pynest/examples/Potjans_2014/network.py:522:62: W504 line break after binary operator
```

Such as:
```
        self.num_neurons = np.round((self.net_dict['full_num_neurons'] *
                                     self.net_dict['N_scaling'])).astype(int)
```

But all of these happen within parentheses, so python will have no problem interpreting them :/ 
The line lengths seem fine :)

I would not force a `\` after each of these binary operators since they are not needed for python to work and make the code cluttered. Maybe the PEP8 checker in Github actions needs to be updated ;)

**Comment by heplesser** [comment]
@morales-gregorio No, we definitely don't want to introduce line continuation by `\`. But PEP8 states that when you break on a binary operator, the operator shall be at the beginning of the second line:
```python
self.num_neurons = np.round((self.net_dict['full_num_neurons']
                                     * self.net_dict['N_scaling'])).astype(int)
```
When fixing this, make sure you don't leave whitespace at the end of the first line or PEP8 will complain about that ;).

**Comment by morales-gregorio** [comment]
Hi! I think all the tests now pass, from my side it can be merged

**Comment by jougs** [comment]
@jhnnsnk: friendly ping!

**Review by jhnnsnk — APPROVED** [review]
:+1: 
Looks good to me. Thanks, @morales-gregorio, for the improvements (and thanks, @jougs, for the ping)!

**Pull request merged**